packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -462,7 +462,7 @@ export class PerformanceEvent {
             this.reportEvent("start");
         }
 
-        if (typeof window === "object" && window != null && window.performance) {
+        if (typeof window === "object" && window != null && window.performance?.mark) {
```

Comment:
Visualizer tests exposed scenario where `window.performance` was defined, but `mark` was not.